src/store/store.js

```diff
@@ -3,7 +3,82 @@ import Vuex from 'vuex';
 
 Vue.use(Vuex);
 
-export default new Vuex.Store({
-  modules: {
+const eventsList = {
+  state: {
+    events: [
+      {
+        id: 0,
```

Comment:
I assume eventually the events are going to need more info (dates, etc.), but this makes sense for now!